.github/stale.yml

```
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+  - blocked
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

I don't think "wontfix" means the same thing as "stale" -- "wontfix" sounds like a refusal to ever consider addressing the issue, whereas "stale" just means it hasn't been prioritized.
```suggestion
staleLabel: stale
```

---

Yeah, true. That also irked me a bit when I copied the config. I think having a dedicated 'stale' label is much better, agreed.